EasyPost.Tests/CalculationTests/RatesTests.cs

```diff
@@ -81,31 +81,31 @@ public void TestLowestRateFilters()
             };
 
             // With no filters, it should go with the first entry with the lowest price
-            Rate lowestRate = Calculation.Rates.GetLowestObjectRate(rates);
+            Rate lowestRate = Utilities.Rates.GetLowestRate(rates);
             Assert.Equal("1.00", lowestRate.Price);
             Assert.Equal("CarrierA", lowestRate.Carrier);
 
             // With a carrier filter, it should go with the first entry with the carrier and lowest price
-            lowestRate = Calculation.Rates.GetLowestObjectRate(rates, new List<string> { "CarrierB" });
+            lowestRate = Utilities.Rates.GetLowestRate(rates, new List<string> { "CarrierB" });
             Assert.Equal("1.00", lowestRate.Price);
             Assert.Equal("CarrierB", lowestRate.Carrier);
 
             // With a service filter, it should go with the first entry with the service and lowest price
-            lowestRate = Calculation.Rates.GetLowestObjectRate(rates, includeServices: new List<string> { "ServiceB" });
+            lowestRate = Utilities.Rates.GetLowestRate(rates, includeServices: new List<string> { "ServiceB" });
             Assert.Equal("1.00", lowestRate.Price);
             Assert.Equal("ServiceB", lowestRate.Service);
 
             // With a carrier and service filter, it should go with the first entry with the carrier and service and lowest price
-            lowestRate = Calculation.Rates.GetLowestObjectRate(rates, new List<string> { "CarrierB" }, new List<string> { "ServiceB" });
+            lowestRate = Utilities.Rates.GetLowestRate(rates, new List<string> { "CarrierB" }, new List<string> { "ServiceB" });
             Assert.Equal("1.00", lowestRate.Price);
             Assert.Equal("CarrierB", lowestRate.Carrier);
 
             // It may go with a higher price if filters are involved
-            lowestRate = Calculation.Rates.GetLowestObjectRate(rates, new List<string> { "CarrierC" });
+            lowestRate = Utilities.Rates.GetLowestRate(rates, new List<string> { "CarrierC" });
             Assert.Equal("100.00", lowestRate.Price);
             Assert.Equal("CarrierC", lowestRate.Carrier);
 
-            lowestRate = Calculation.Rates.GetLowestObjectRate(rates, includeServices: new List<string> { "ServiceC" });
+            lowestRate = Utilities.Rates.GetLowestRate(rates, includeServices: new List<string> { "ServiceC" });
```

Comment:
Oof, we were using the internal interface and not the user-facing one heh. Glad this is fixed.

Reply:
To be fair, there was no user-facing one (or at least, "Calculation" was the user-facing one).